src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -114,7 +122,10 @@ export default {
 			if (this.isConnecting) {
 				return t('spreed', 'Connecting …')
 			}
-			return t('spreed', 'Waiting for others to join the call …')
+			if (this.isOneToOneConversation()) {
```

Comment:
Error:   125:8   error  Use this.isOneToOneConversation instead of this.isOneToOneConversation()  vue/no-use-computed-property-like-method